frontend/app/(authenticated)/messages/_components/UserItem.tsx

```diff
@@ -17,7 +17,7 @@ const UserItem: React.FC<UserItemProps> = ({
 
   return (
     <div className="w-full border-b border-gray-300 p-4">
-      <div onClick={onClick} className="flex items-center gap-x-4 cursor-pointer">
+      <div onClick={onClick} className="flex items-center gap-x-4 hover:bg-gray-50 cursor-pointer rounded">
```

Comment:
As I texted you, add the style to `div` in line 19!

Since we want to apply the gray bg color to a whole `UserItem`, you don't need `rounded`